src/renderer/components/menu.vue

```diff
@@ -1,22 +1,18 @@
 <template>
-    <div class="top-bar">
-        <div class="top-bar-left">
-            <ul class="menu">
-              <router-link tag="li" active-class="is-active" :to="{ name: 'home-page' }">
-                <a><i class="fas fa-home"></i>&nbsp;Home</a>
-              </router-link>
-              <router-link tag="li" active-class="is-active" :to="{ name: 'settings' }">
-                <a><i class="fas fa-cogs"></i>&nbsp;Settings</a>
-              </router-link>
-              <router-link tag="li" active-class="is-active" :to="{ name: 'about-page' }">
-                <a><i class="fas fa-info-circle"></i>&nbsp;About</a>
-              </router-link>
-            </ul>
-        </div>
-        <div class="top-bar-right">
-          <ul class="menu">
-            <li><a @click="event => openSite(event, this.helpLink)"><i class="fas fa-question-circle"></i>&nbsp;Help</a></li>
-          </ul>
+    <div class="ui huge secondary menu">
```

Comment:
Please use a page menu as described in the [additional features](https://github.com/FirstLegoLeague/user-interface/blob/master/docs/additional-features.md) page and the [UI standard](https://github.com/FirstLegoLeague/user-interface/blob/master/docs/UI-standard.md) page. If you need an example for the correct way to use it you can see in any of the pedning semantic UI branches.